lib/govuk_ab_testing/rspec_helpers.rb

```diff
@@ -0,0 +1,39 @@
+module GovukAbTesting
```

Comment:
Yes possibly - I thought about this a bit more and maybe you could move all the setup and assertion steps into `active_support.rb` and `capybara.rb`? Then you could just have a single helper file (or two very lightweight helpers that point to a common one with different initialization). That would avoid the need for another layer of abstraction, and just slightly move the boundary of abstraction in a way that will keep the testing logic consistent.